source/develop/release-management/features/virt/high-performance-vm.html.md

```diff
@@ -93,60 +91,67 @@ Displayed in 'Resource Allocation' side-tab of the VM dialog.
 
-In case the I/O Thread pinning topology is not set upon VM saving,  a recommendation/warning will be displayed in a pop-up and the user will be asked to pin the I/O thread to a host and verify it fits the host configuration.
+In case the I/O Thread pinning topology is not set upon VM saving, a recommendation/warning will be displayed in a pop-up and the user will be asked to pin the I/O thread to a host and verify it fits the host configuration.
 Consider verifying if the  I/O is bound to specific CPU on host level and chose that CPU for pinning the I/O Thread.
 
+In addition, we will suggest to set the emulatorpin to be pinned to the same CPU(s) as the I/O Thread. Should consider how the user will do that (using a hook or via VDSM).
+
 #### **Enable virtual NUMA and set NUMA pinning topology**
 Displayed in 'Host' side-tab of the VM dialog.
 
-In case the virtual NUMA and the NUMA pinning topology are not set upon VM saving,  a recommendation/warning will be displayed in a pop-up and the user will be asked to pin the VM to a host that supports NUMA topology and verify the VM NUMA topology fits the host's NUMA topology.
+In case the virtual NUMA and the NUMA pinning topology are not set upon VM saving, a recommendation/warning will be displayed in a pop-up and the user will be asked to pin the VM to a host that supports NUMA topology and verify the VM NUMA topology fits the host's NUMA topology.
 
 Ideally the NUMA pinning should be done automatically according to the exposed host's NUMA topology and not manually by the user, but in first phase we will not support it.
 
 #### **disable kernel same page merging (KSM)**
 This is currently implemented in engine only for cluster level (new/edit cluster dialog->'Optimization side-tab). 
 
 KSM can be manually deactivated in host level by stopping the ksmtuned and the ksm service on the hypervisor.
-Disabling the KSM for the specific VM should be implemented by adding a check-box to the 'Resource Allocation' side-tab of the VM dialog and implemented by setting the "nosharepages" domain property in libvirt xml.
 
-In case that KSM is not disabled upon VM saving,  a recommendation/warning will be displayed in a pop-up and the user will be asked to disable it either in cluster level, host level or for the specific VM.
+Disabling the KSM for the specific VM may be implemented for oVirt 4.3 if required. This should be implemented by adding a check-box to the 'Resource Allocation' side-tab of the VM dialog and implemented by setting the "nosharepages" domain property in libvirt xml (requires  VDSM changes).
+
+In case that KSM in cluster level or host level is not disabled upon VM saving, a recommendation/warning will be displayed in a pop-up and the user will be asked to disable it either in cluster level, host level or for the specific VM.
 
 #### **Enable  memory backing with huge pages**
 This is currently not implemented. 
```

Comment:
Everything required for the dialog is already merged, we only lack scheduler awareness.